nova/api/openstack/compute/routes.py

```diff
@@ -848,6 +848,9 @@ def _create_controller(main_controller, action_controller_list):
     ('/servers/{server_id}/topology', {
         'GET': [server_topology_controller, 'index']
     }),
+    ('/sap/usage/{id}', {
+        'GET': [sap_admin_api_controller, 'show_usage'],
+    }),
```

Comment:
more FYI: I would have re-used the exising `/sap/{action}` and called it something like `limes_usage` or `usage_by_az`, getting the project_id from the query strings instead of the URL itself.

Reply:
I thought it wasn't very openstack-like, but I changed it now to a query parameter.